commit 9a3428d6d98cc9c2a06af00feb518c85fd45c4c7 (refs/changes/92/2992/4)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2018-01-30T19:22:22+00:00 (1 year, 8 months ago)
    
    TRITON-18 vuln in semver module
    Reviewed by: Julien Gilli <julien.gilli@joyent.com>
    Approved by: Julien Gilli <julien.gilli@joyent.com>

diff --git a/package.json b/package.json
index c789b8c..3ed7067 100644
--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     "optimist": "0.3.5",
     "restify": "2.4.1",
     "sdc-clients": "9.2.0",
-    "semver": "3.0.1",
+    "semver": "5.4.1",
     "sprintf-js": "0.0.7",
     "vasync": "^1.6.3",
     "verror": "^1.10.0"
